datafusion/core/src/physical_optimizer/combine_partial_final_agg.rs

```diff
@@ -362,7 +362,7 @@ mod tests {
         let expected = &[
             "AggregateExec: mode=Final, gby=[], aggr=[COUNT(2)]",
             "AggregateExec: mode=Partial, gby=[], aggr=[COUNT(1)]",
-            "ParquetExec: limit=None, partitions={1 group: [[x]]}, projection=[a, b, c]",
+            "ParquetExec: file_groups={1 group: [[x]]}, projection=[a, b, c], limit=None, output_ordering=[]",
```

Comment:
Would it be possible to keep the existing behavior and only print out `limit`, `projection` and `output_ordering` if they were non empty?

So like in this case it would be

```
file_groups={1 group: [[x]]}, projection=[a, b, c]",
```

Because `limit` is `None` and `output_ordering` is empty?

The rationale is to keep the display easier to read by keeping the output concise

Reply:
Yup just now I second that a compact output will be better.

Made the change accordingly!